app/src/routes/login/login.vue

```diff
@@ -78,7 +79,11 @@ export default defineComponent({
 			providerOptions.value = providers.value
 				.filter((provider) => !AUTH_SSO_DRIVERS.includes(provider.driver))
 				.map((provider) => ({ text: formatTitle(provider.name), value: provider.name }));
-			providerOptions.value.unshift({ text: t('default_provider'), value: DEFAULT_AUTH_PROVIDER });
+			if (!disableDefault.value) {
+				providerOptions.value.unshift({ text: t('default_provider'), value: DEFAULT_AUTH_PROVIDER });
+			} else {
+				providerSelect.value = providerOptions.value[0].value;
```

Comment:
Is this actually required? I think v-select should just pick up the first from `providerOptions`.

Reply:
Yes, I whink so. I'm not really sure why, but without it, the select defaults to "local" without it actually being a selectable option.

Reply:
Yes, I think so. I'm not really sure why, but without it, the select defaults to "local" without it actually being a selectable option.

Reply:
Ah, yes. Because it uses the `provider.value` which is `DEFAULT_AUTH_PROVIDER`. 👍